api/_mixins/ParentNode__Element.json

```
@@ -438,7 +438,7 @@
               "version_added": "14"
             },
             "safari_ios": {
-              "version_added": "14"
+              "version_added": "14.5"
```

```suggestion
              "version_added": "14"
```

As above, but tested with https://mdn-bcd-collector.appspot.com/tests/api/Element/replaceChildren